### ISSUE TITLE
If an item is included in a time selection, the next item does not play and playback remains in the time selection

### ISSUE THREAD

**Issue by Lo-lo78** [jcsteh/osara#1044]
Hi, if I select an item and insert a time selection with the action:
"Time selection: Set time selection to items"
the next item is selected but does not play.

Example 1:
Item 4 selected:
sets the time selection on the item,
right arrow,
Item 5 selected:
the tempo selection corresponding to item 4 is played.
Not expected.
Item 6 selected:
plays as expected.
Left arrow,
Item 5 selected:
the time selection of item 4 is played.
Not expected.
Left arrow,
item 4 selected:
the time selection and item 4 are played.
As expected.
Left arrow,
Item 3 selected:
plays as expected.

Example 2:
Item 5 selected:
the time selection of item 4 is played.
Not expected.
Item 5 selected:
escape key,
exits the selection and item 5 goes into play.
As expected.

**Comment by ScottChesworth** [comment]
Do you have REAPER stopping at the end of loop if repeat is disabled? Check Preferences, Playback category. If that's checked at the moment, disable it and re-test.

**Comment by Lo-lo78** [comment]
It works as expected.
Goes to the next item.
but "repeat on" I use it 90% of the time.

**Comment by Lo-lo78** [comment]
@ScottChesworth 
If the action can be useful:
"Go to end of time selection"
in this case it does not select item 5 of the example but goes to the beginning of the unselected item 5 and is played as expected.

**Comment by Lo-lo78** [comment]
It's definitely a Reaper bug.
Let's hear from James and if so we'll close the issue.

**Comment by Lo-lo78** [comment]
@ScottChesworth 
Yes. Now I've tried both preferences and actions.
If it is in of:
"Transport: Toggle repeat"
and is also enabled:
"Transport: Toggle stop playback at end of loop if repeat is disabled".
If I select the item and create a time selection, the next item does not go into play even if it is stopped.
Really not expected.
You need to do the escape key or action:
"Go to end of time selection"
to exit the selection and be in play.
In summary, the next item if the previous one is selected has problems.

**Comment by jcsteh** [comment]
I don't entirely understand the steps to reproduce here. However, if I set a time selection to one item and then move to the next item, as long as the next item is outside the time selection, playback works as expected, even with stop at the end of loop if repeat is disabled turned on. The only reason playback would stop is if your next item is still inside the time selection, which I guess can happen if your items overlap, but then this is definitely expected behaviour, since the whole point of that toggle is to stop playback at the end of the loop area.

In any case, this is definitely not anything OSARA has any control over.

**Comment by Lo-lo78** [comment]
@jcsteh 
If you use the action:
"Time selection: Set time selection to items"
The next item has strange behavior in my Reaper.
Thanks for the reply.
